main.go

```diff
@@ -156,8 +157,12 @@ func main() {
 		minioInformerFactory = informers.NewSharedInformerFactory(controllerClient, time.Second*30)
 		promInformerFactory = prominformers.NewSharedInformerFactory(promClient, time.Second*30)
 	}
+	podName := os.Getenv("POD_NAME")
```

Comment:
```suggestion
	podName := os.Getenv("HOSTNAME")
```

We can get the pod name via the `HOSTNAME` env variable

Reply:
done